src/OmniSharp.Roslyn/MetadataHelper.cs

```diff
@@ -80,7 +80,7 @@ public Task<Document> GetDocumentFromMetadata(Project project, ISymbol symbol, C
 
         public async Task<Location> GetSymbolLocationFromMetadata(ISymbol symbol, Document metadataDocument, CancellationToken cancellationToken = new CancellationToken())
         {
-            var symbolKeyCreateMethod = _symbolKey.Value.GetMethod("Create", BindingFlags.Static | BindingFlags.NonPublic);
+            var symbolKeyCreateMethod = _symbolKey.Value.GetMethod("Create", BindingFlags.Static | BindingFlags.Public);
```

Comment:
Note: This was the only change needed to allow the metadata-as-source reflection to continue working.